nlplug-findfs.c

```diff
@@ -934,8 +954,12 @@ static int searchdev(struct uevent *ev, const char *searchdev, int scanbootmedia
 			start_mdadm(ev->devnode);
 		} else if (strcmp("LVM2_member", type) == 0) {
 			start_lvm2(ev->devnode);
+		} else if (strcmp("zfs_member", type) == 0) {
+			start_zpool(ev->devnode);
 		} else if (scanbootmedia) {
 			rc = scandev(conf, ev->devnode, type);
+		} else if (bootrepos) {
+			rc = find_bootrepos(ev->devnode, type, bootrepos, apkovls);
```

Comment:
This hunk needs to be removed. `find_bootrepos` got renamed to `scandev` and `bootrepos` got replaced by `scanbootmedia` in 7673ee7d1ce6786a5471ead15513d62027c9756a